lib/task/i18n/i18nRemoveHtmlTagsTask.class.php

```diff
@@ -128,23 +179,23 @@ public function execute($arguments = array(), $options = array())
    *
    * @return integer  number of columns changed
    */
-  private function processInformationObjectI18nHtml(&$io)
+  private function processInformationObjectI18nHtml(&$io, $tableName, $columns)
```

Comment:
Can we rename this function since it no longer is specific to information objects?